docs/core/tools/dotnet-test.md

```diff
@@ -5,13 +5,33 @@ ms.date: 03/27/2024
 ---
 # dotnet test
 
-**This article applies to:** ✔️ .NET Core 3.1 SDK and later versions
-
 ## Name
 
 `dotnet test` - .NET test driver used to execute unit tests.
 
-## Synopsis
+## Description
+
+The `dotnet test` command builds the solution and runs the tests either with VSTest or with Microsoft Testing Platform (MTP). In order to enable MTP, you need to add a config file named `dotnet.config` with TOML format located at the root of the solution or repository.
+
+Some examples of the `dotnet.config` file:
+
+  ```toml
```

Comment:
It looks like `toml` is not a valid language tag: https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies/devlang?branch=main#valid-values. However, [we also use `ini` elsewhere in the docs](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0220#suppress-a-warning), and I see that isn't valid either, so I think it's okay to keep this.